perception/compare_map_segmentation/README.md

```diff
@@ -75,13 +75,14 @@ WIP
 
 #### Parameters
 
-| Name                            | Type  | Description                                                                                  | Default value |
-| :------------------------------ | :---- | :------------------------------------------------------------------------------------------- | :------------ |
-| `use_dynamic_map_loading`       | bool  | map loading mode selection, `true` for dynamic map loading, `false` for static map loading   | true          |
-| `distance_threshold`            | float | VoxelGrid's leaf_size also the threshold to check distance between input point and map point | 0.5           |
-| `map_update_distance_threshold` | float | Threshold of vehicle movement distance when map update is necessary [m]                      | 10.0          |
-| `map_loader_radius`             | float | Radius of map need to be loaded (in dynamic map loading) [m]                                 | 150.0         |
-| `timer_interval_ms`             | int   | time interval of timer to check if update the map is necessary (in dynamic map loading) [ms] | 100           |
+| Name                            | Type  | Description                                                                                                                             | Default value |
+| :------------------------------ | :---- | :-------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| `use_dynamic_map_loading`       | bool  | map loading mode selection, `true` for dynamic map loading, `false` for static map loading                                              | true          |
+| `distance_threshold`            | float | VoxelGrid's leaf_size also the threshold to check distance between input point and map point                                            | 0.5           |
```

Comment:
I can not understand the meaning of the description here.
Where does "VoxelGrid's leaf_size" come from?

Reply:
Thank you for your comment, I corrected here https://github.com/autowarefoundation/autoware.universe/pull/3277/commits/8d7c4d4e08f55bb58e515ea2ef3fd2bd6378089f